dali/kernels/imgproc/resample/resampling_setup.cc

```diff
@@ -208,6 +212,11 @@ void SeparableResamplingSetup<2>::ComputeBlockLayout(SampleDesc &sample) const {
  */
 template <>
 void SeparableResamplingSetup<3>::ComputeBlockLayout(SampleDesc &sample) const {
+  if (volume(sample.out_shape()) == 0) {
```

Comment:
Why there is no similar change for `SeparableResamplingSetup<2>`?

Reply:
because the 2D version does not contain division - it will end up with zeros in logical_block_shape anyway.

Reply:
:+1: